schedules/your_workflows.py

```diff
@@ -0,0 +1,18 @@
+from datetime import timedelta
+
+from temporalio import workflow
+from your_dataobject import YourParams
```

Comment:
I think this can fit under `your_activities` since it's only activity params. Even if you do keep it in its own files, might as well pass it through also.